src/components/Form.js

```diff
@@ -148,6 +148,20 @@ class Form extends React.Component {
                 });
             }
 
+            // Drill down the component that have a role as form wrapper
+            // e.g usage of AddressForm or IdentityForm
+            if (_.isFunction(child.type)) {
+                const nestedChildren = new child.type(child.props);
+
+                if (!React.isValidElement(nestedChildren) || !nestedChildren.props.children) {
```

Comment:
Are we always sure we'll have `nestedChildren.props`? I am wondering if this needs to be replaced with `lodash.get` ?

Reply:
Good thinking! I'm not really sure but let's use `lodash.get` instead.